src/components/CameraError/index.js

```diff
@@ -58,11 +58,7 @@ export default class CameraError extends Component<Props, State> {
           onDismiss={this.handleDismiss}
           renderInstruction={ str => parseTags(str,
             ({text}) =>
-            <button
-              type="button"
-              onClick={this.handleFallbackClick}
-              className={style.fallbackLink}
-            >
+            <button type="button" onClick={this.handleFallbackButtonClick} className={style.fallbackButton} >
```

Comment:
Seems like the button cannot be clicked using the Enter key. Not sure why. Maybe it needs an event handler for the enter key.

Reply:
Have fixed this by refactoring the "button within a button" implementation here to use a common `FallbackButton` component instead. I've put this component under the `Button` folder since that seemed like the best place to me.